test/specs/css-variables/spec.svg

```diff
@@ -0,0 +1,14 @@
+<svg viewBox="0 0 300 150" xmlns="http://www.w3.org/2000/svg">
+  <style>.bluePolyline {stroke: var(--color); } .polyLineColors {--color:  blue;}</style>
+  <style>.redPolyline { --colorRed: red; stroke: var(--colorRed);}</style>
+  <style>.greenPolyline { stroke: var(--colorGreen);}</style>
```

Comment:
Just nitpicking, but "textColors" would probably be more understandable than "polylineColors" here.

Reply:
Yep , I just copy code from issues. not rename class.
It think Css variable is used by current element.
But, Css variable's key-value definition can in anywhere.
So, when I get ` - - color`lookup for a variable value includes all available CSS rule.

Reply:
I haven't checked the code too much,
I don't know if I can use the DOM API to get the current element style or analytic CSS-variables key,
if there is no DOM, maybe I can only get the CSS-variables key through Regular Expression.

Reply:
You can use the DOM API.